Chap_API_Proc_Mgmt.tex

```diff
@@ -356,15 +372,15 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_HOSTFILE}{"pmix.hostfile"}{char*}{
-Hostfile to use for spawned processes.
+Hostfile to use for spawned processes.  The format of this file is determined by the \ac{PMIx} implementation or host environment.  The file may not be portable across implementations.
```

Comment:
Nothing to do with the PMIx implementation - we just pass the name of the file.

Reply:
This comes down to whether we want to force the division of labor between PMIx or the host environment.  We didn't want to preclude the possibility that someone say, took OpenPMix and decided to implement spawn directly inside of it.   If we think that is never going to happen, we can go with:

The format of this file is determined by the host environment.  A hostfile may not be portable across systems with different host environments.

Reply:
We do support local fork/exec from within the PMIx library, but that is nowhere equivalent to spawn as we provide no support for launching things across nodes. We really should stick with the divisions we have had since day one and not muddy the water.